jaxrs/src/test/scala/com/cerner/beadledom/jaxrs/PATCHSpec.scala

```diff
@@ -0,0 +1,77 @@
+package com.cerner.beadledom.jaxrs
+
+import com.cerner.beadledom.jaxrs.provider.{FakeModel, FakeResource}
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider
+
+import org.jboss.resteasy.core.Dispatcher
+import org.jboss.resteasy.mock.{MockDispatcherFactory, MockHttpRequest, MockHttpResponse}
+import org.jboss.resteasy.plugins.server.resourcefactory.POJOResourceFactory
+import org.scalatest._
+import org.scalatest.mock.MockitoSugar
+
+import java.io._
+
+import scala.collection.JavaConverters._
+import javax.ws.rs.core.MediaType
+
+/**
+  * Spec tests for {@link PATCH}.
+  *
+  * @author Eric Christensen
+  */
+class PATCHSpec extends FunSpec with BeforeAndAfterAll with ShouldMatchers with MockitoSugar {
```

Comment:
PATCH used with an actual stood up service in the "retrieves the resources from different clients test" [d582c3](https://github.com/cerner/beadledom/pull/36/commits/d582c3a78cc5f6ca53ed2ee53d7c9507bdb69565)